docs/components/dividers.md

```diff
@@ -0,0 +1,225 @@
+<!--docs:
+title: "Material dividers"
+layout: detail
+section: components
+excerpt: "A divider is a thin line that groups content in lists and layouts."
+iconId:
+path: /catalog/material-dividers/
+-->
+
+# Dividers
+
+A [divider](https://material.io/components/dividers) is a thin line that groups content in lists and layouts.
+
+![Example divider: full-bleed dividers](assets/dividers/Dividers_hero.png)
+
+## Contents
+
+- [Contents](#contents)
```

Comment:
I thought we removed the content line from the contents

Reply:
Done